js/console.ts

```diff
@@ -351,6 +351,9 @@ export function stringifyArgs(
 
 type PrintFunc = (x: string, isErr?: boolean) => void;
 
+const countMap = new Map<string, number>();
+const timerMap = new Map<string, number>();
```

Comment:
should these be in the Console class?

Reply:
js has no private property (currently), if it is placed inside the `Console` class, these variables are accessed by by code outside the class. And these properties will be printed with console.

currently The `printFunc` method is rinted and can be used by external code. I think this is a wrong implementation, and this should be refactored in the future.

```shell
> console
Console { printFunc: [Function], log: [Function], debug: [Function], info: [Function], dir: [Function], warn: [Function], error: [Function], assert: [Function] }
> console.printFunc
[Function]
> console.printFunc.toString()
function () { [native code] }
> console.printFunc('hello')
hello
undefined
```